spec/requests/admin/email_controller_spec.rb

```diff
@@ -84,6 +84,23 @@
         expect(logs.first["reply_key"]).to eq(post_reply_key_2.reply_key)
       end
     end
+
+    it 'should be able to filter by smtp_transaction_response' do
+      email_log_2 = Fabricate(:email_log, smtp_transaction_response: <<~RESPONSE)
+250 Ok: queued as pYoKuQ1aUG5vdpgh-k2K11qcpF4C1ZQ5qmvmmNW25SM=@mailhog.example
+      RESPONSE
```

Comment:
```suggestion
      email_log_2 = Fabricate(:email_log, smtp_transaction_response: <<~RESPONSE)
      250 Ok: queued as pYoKuQ1aUG5vdpgh-k2K11qcpF4C1ZQ5qmvmmNW25SM=@mailhog.example
      RESPONSE
```

Note that when you use the squiggly heredoc (`<<~`), you can continue to indent your code as per normal and all leading spaces are ignored.